tests/phpunit/FlattenFilenameTest.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\Processor\FlattenFolders\Tests;
+
+/**
```

Comment:
Můžeš si ty docblocky rovnou vyhodit z těch file templates ;)

Reply:
to byla poctivá ruční práce! :D